src/Uno.UI.Runtime.Skia.Wpf/UI/Controls/UnoWpfWindow.cs

```diff
@@ -95,6 +98,24 @@ internal void UpdateWindowPropertiesFromApplicationView()
 		MinHeight = appView.PreferredMinSize.Height;
 	}
 
+	internal void UpdateWindowPropertiesFromCoreApplication()
+	{
+		var coreApplicationView = CoreApplication.GetCurrentView();
+		if (coreApplicationView.TitleBar.ExtendViewIntoTitleBar)
+		{
+			WindowStyle = WindowStyle.None;
+			WindowChrome.SetWindowChrome(this, new WindowChrome()
+			{
+				GlassFrameThickness = new Thickness(-1),
```

Comment:
Consider update the `CaptionHeight` property to keep the same bahavior as https://github.com/unoplatform/uno/pull/16202